code/test/user.donation.test.js

```diff
@@ -0,0 +1,152 @@
+const expect = require('chai').expect;
+const mongoose = require('mongoose');
+const request = require('supertest');
+const sinon = require('sinon');
+const sandbox = sinon.createSandbox();
+
+/*
+  These will be included on a case-by-case basis
+*/
+const app = require('../app'); // used for supertest
+const donationCodes = require('../config/resCodes/user/donation');
+const testVar = require('../config/testVar'); // generic test donation
+const Donation = require('../models/donation');
+
+const donationFunctions = require('../routers/user/donation');
+
+describe('Donation functions', function() {
+  
+  describe('Create Donation Test', function() {
+
+    before(function(done) {
+
+      /*
+        Create any stubs or variables you need for the tests
+      
+        example stub:
+        var functionStub = sandbox.stub([object to stub], [function to stub])
+        functionStub.yields - used to insert values into callback function 
+        functionStub.onCall(n).yields - onCall is used to change the behavior of the stub the nth time the functon is called ( see admin.test.js )
+        functionStub.withArgs([args]).yields - withArgs is uesed to cahnge the bahvior of the stub when called with specific arguments ( see user.models.test.js )
+      */
```

Comment:
Delete this. Same as above. Anything from the example test should not be included unless it is needed/used